src/main/java/gregtech/common/metatileentities/multi/steam/MetaTileEntitySteamBlastFurnace.java

```diff
@@ -0,0 +1,73 @@
+package gregtech.common.metatileentities.multi.steam;
+
+import gregtech.api.capability.impl.SteamMultiWorkable;
+import gregtech.api.metatileentity.MetaTileEntity;
+import gregtech.api.metatileentity.MetaTileEntityHolder;
+import gregtech.api.metatileentity.multiblock.IMultiblockPart;
+import gregtech.api.metatileentity.multiblock.MultiblockAbility;
+import gregtech.api.metatileentity.multiblock.RecipeMapSteamMultiblockController;
+import gregtech.api.multiblock.BlockPattern;
+import gregtech.api.multiblock.FactoryBlockPattern;
+import gregtech.api.recipes.RecipeMaps;
+import gregtech.api.render.ICubeRenderer;
+import gregtech.api.render.OrientedOverlayRenderer;
+import gregtech.api.render.Textures;
+import gregtech.common.ConfigHolder;
+import gregtech.common.blocks.BlockMetalCasing;
+import gregtech.common.blocks.MetaBlocks;
+import net.minecraft.block.state.IBlockState;
+import net.minecraft.util.ResourceLocation;
+
+import javax.annotation.Nonnull;
+
+import static gregtech.api.render.Textures.BRONZE_PLATED_BRICKS;
+import static gregtech.api.render.Textures.SOLID_STEEL_CASING;
+
+public class MetaTileEntitySteamBlastFurnace extends RecipeMapSteamMultiblockController {
+
+    private static final MultiblockAbility<?>[] ALLOWED_ABILITIES = {
+            MultiblockAbility.STEAM_IMPORT_ITEMS, MultiblockAbility.STEAM_EXPORT_ITEMS, MultiblockAbility.STEAM
+    };
+
+    public MetaTileEntitySteamBlastFurnace(ResourceLocation metaTileEntityId) {
+        super(metaTileEntityId, RecipeMaps.STEAM_BLAST_FURNACE_RECIPES, CONVERSION_RATE);
+        this.recipeMapWorkable = new SteamMultiWorkable(this, CONVERSION_RATE, 4);
```

Comment:
This needs fixing after Trouser's Parallel PR was merged in, and also needs to set the parallel limit in the new way after Tech's PR.
`this.recipeMapWorkable.setParallelLimit(4)`

Reply:
Fixed